models/tasks.js

```diff
@@ -284,6 +284,12 @@ const overdueTasks = async (overDueTasks) => {
     throw err;
   }
 };
+
+const generateTaskNo = async () => {
+  const snapshot = await tasksModel.count().get();
+  return snapshot.data().count + 1;
+};
```

Comment:
Thinking out loud:  Is there a better way to number tasks? because for the task that is not there anymore, a number assigned to it gets wasted.

Reply:
`the task that is not there anymore`. Can you provide some more clarity on it?